onnxruntime/core/optimizer/gemm_sum_fusion.cc

```diff
@@ -0,0 +1,161 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "core/optimizer/gemm_sum_fusion.h"
+
+#include "core/graph/graph_utils.h"
+#include "core/optimizer/initializer.h"
+#include "core/optimizer/utils.h"
+
+using namespace ONNX_NAMESPACE;
+using namespace onnxruntime::common;
+namespace onnxruntime {
+
+Status GemmSumFusion::Apply(Graph& graph, Node& gemm_node, RewriteRuleEffect& modified, const logging::Logger&) const {
+  // Get currently set attributes of Gemm. Beta will become 1.0.
+  bool transA = static_cast<bool>(gemm_node.GetAttributes().at("transA").i());
```

Comment:
Let's explicitly add `const` for all constants, for example,
```suggestion
  const bool transA = static_cast<bool>(gemm_node.GetAttributes().at("transA").i());
```